metron-interface/metron-rest/src/main/java/org/apache/metron/rest/service/impl/StormStatusServiceImpl.java

```diff
@@ -49,6 +44,11 @@ public StormStatusServiceImpl(Environment environment, RestTemplate restTemplate
   }
 
   @Override
+  public SupervisorSummary getSupervisorSummary(){
+    return restTemplate.getForObject("http://" + environment.getProperty(STORM_UI_SPRING_PROPERTY) + SUPERVISOR_SUMMARY_URL, SupervisorSummary.class);
```

Comment:
Should have pointed this out the other day, but that PR is in now, so this probably should be:
```
return restTemplate.getForObject(getStormUiProperty() + SUPERVISOR_SUMMARY_URL, SupervisorSummary.class);
```

Ping me if there's any issues and I'm happy to do more than just glance at it.